src/tests/topp/FalseDiscoveryRate_5_output.idXML

```diff
@@ -5,7586 +5,6 @@
 	</SearchParameters>
 	<IdentificationRun date="2015-06-29T15:12:34" search_engine="Fido" search_engine_version="" search_parameters_ref="SP_0" >
 		<ProteinIdentification score_type="q-value" higher_score_better="false" significance_threshold="0" >
```

Comment:
why does this remove all Protein ids? Is this intended?

Reply:
I think it is because in the input (FDR_output_1.idXML), all proteins have a score of 0. So the FDR for all will be #decoys / #all which is probably higher than the default cutoff. I think that was the reason for Timo's comment.

Reply:
does this make sense? I mean the q-value should be the minimal FDR so these should not be removed

Reply:
I think the FDR calculation does not care about the type of score that is in there. It uses what is in there and calculates the FDR anew.

Reply:
It is weird that a dataset with already annotated q-values is used for FDR calculation AT ALL. That is what we want to compute here.